examples/timer/timer.mk

```diff
@@ -39,6 +39,11 @@ CLIENT_OBJS := client.o
 TIMER_DRIVER := $(SDDF)/drivers/clock/$(TIMER_DRIVER_DIR)
 
 all: $(IMAGE_FILE)
+CHECK_FLAGS_BOARD_MD5:=.board_cflags-$(shell echo -- ${CFLAGS} ${BOARD} ${MICROKIT_CONFIG} | shasum | sed 's/ *-//')
```

Comment:
Shouldn't this file be a dependency of some object files?

Reply:
It's used in the snippets.  Ideally, we'd have a separate common.mk somewhere to include that sets this, and some general rules.  So right now, I'm generating it in the top-level makefiles.